apps/examples/mediaplayer/mediaplayer_main.cpp

```diff
@@ -338,12 +369,87 @@ class MediaPlayerController
 };
 
 extern "C" {
+
+static void wifi_sta_connected(wifi_manager_result_e result) {
+	printf("[%s] result %d\n", __FUNCTION__, (int)result);
+}
+
+static void wifi_sta_disconnected(wifi_manager_disconnect_e result) {
+	printf("[%s] result %d\n", __FUNCTION__, (int)result);
+}
+
+static wifi_manager_cb_s wifi_callbacks = {
+	wifi_sta_connected,
+	wifi_sta_disconnected,
+	NULL,
+	NULL,
+	NULL,
+};
+
+static int wifi_connect()
+{
+	wifi_manager_result_e ret = WIFI_MANAGER_FAIL;
+	wifi_manager_ap_config_s config;
+
+	/**
+	 * Need to sleep for WiFi initialized.
+	 */
+	int i = 0;
+	for (i = 0; i < 3; i++) {
```

Comment:
I do not need this logic in my opinion.
Why was it added?

Reply:
It was history code from other example.
Removed now.